### ISSUE TITLE
Redirect missing requests to default status page

### ISSUE THREAD

**Issue by ToshY** [tarampampam/error-pages#188]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the problem to be solved

Use case:

I require 404 status codes to be handled by the application instead of error-pages. To do so I now have the following middleware setup in Traefik config.

```yaml
http:
  middlewares:
    errorPages:
      errors:
        # 404 excluded
        status: 400-403,405-499,501,502,504-599
        service: error-pages@docker
        query: /{status}.html
```

Error-pages is setup with docker compose:

```yaml
version: '3.9'

services:
  error-pages:
    image: tarampampam/error-pages:${ERROR_PAGES_IMAGE_VERSION} # 2.21
    security_opt:
      - no-new-privileges:true
    restart: always
    environment:
      TEMPLATE_NAME: ${ERROR_PAGES_TEMPLATE_NAME} # cats
      SHOW_DETAILS: ${ERROR_PAGES_SHOW_DETAILS} # true
    labels:
      traefik.enable: true
      traefik.docker.network: proxy
      traefik.http.routers.error-pages.tls: true
      traefik.http.routers.error-pages.rule: HostRegexp(`{host:.+}`)
      traefik.http.routers.error-pages.priority: 10
      traefik.http.routers.error-pages.entrypoints: websecure
      traefik.http.routers.error-pages.service: error-pages
      traefik.http.services.error-pages.loadbalancer.server.port: 8080
    networks:
      - error-pages
      - proxy
 ```

Now the following currently happens:

* If `example.com` is up and a user requests a non-existing page, e.g. `example.com/test`, the user will see the application's specific error page.
* If `example.com` is down it shows the 404 (cats) error-pages when requesting `example.com`. But requesting a non-existing page, e.g. `example.com/test`, the user will now see the following message:

```yaml
Wrong request URL. Error pages are available at the following URLs: /{code}.html
```

What I would like to see is that it defaults to the default error page (cats 404) instead of this message.

### Suggest a solution

Possible options:

* Add a new option/variable to specify to fallback on the `404.html` page when at the  [notfound handler](https://github.com/tarampampam/error-pages/blob/master/internal/http/handlers/notfound/handler.go).
* Use the existing `DEFAULT_ERROR_PAGE`  / `DefaultErrorPage` variable and use it at the [notfound handler](https://github.com/tarampampam/error-pages/blob/master/internal/http/handlers/notfound/handler.go).

> Note: I'm not familiar with Go.

### Additional context

* This was also mentioned in #140

---

* Docker Compose v2.17
* Traefik v2.9
* Error-pages 2.21

**Comment by tarampampam** [comment]
Hi @ToshY! Is that what u need? Could check the changes on your side? https://github.com/tarampampam/error-pages/pull/189

**Comment by tarampampam** [comment]
You can use the following image tag for this: `tarampampam/error-pages:issue-188`

**Comment by ToshY** [comment]
@tarampampam Works like a charm 👍

**Comment by tarampampam** [comment]
🔥 Released in [v2.22.0](https://github.com/tarampampam/error-pages/releases/tag/v2.22.0)
